pkg/kapp/cmd/appchange/sort_flag.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2020 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package appchange
+
+import "github.com/spf13/cobra"
+
+type SortFlag struct {
+	Sort string
+}
+
+func (s *SortFlag) Set(cmd *cobra.Command) {
+	cmd.Flags().StringVarP(&s.Sort, "sort", "s", "newest-first", "Sort app-changes list (allowed values : newest-first, oldest-first)")
```

Comment:
What do you think about using a boolean flag called `newest-first` with default set to true?
Also, since the sorting is happening based on just the time field, do we want to keep this with `time_flags` itself?

Reply:
Yeah can certainly we can do that but just `--newest-first` by name is not indicating that we gonna sort by newest first. We can discuss this further and get thoughts from others to. 

I think if we decide to go just with the boolean flag then it make sense to add it in `time_flags`

Reply:
I have been thinking how we can avoid a string flag, so it makes sense to do that. I wonder if we could name it better tho (`--newest-first false`  does not _necessarily_ indicate oldest first). I think newest-first is alright but worth some thought.

Reply:
+1 on merging sort flag into time flags